src/surfaces/drawer/drawerDataGroups.d.ts

```diff
@@ -0,0 +1,54 @@
+import { string } from 'prop-types';
+import React from 'react';
+
+export interface DrawerDataGroupsPropTypes {
+    bleed?: boolean;
+    className?: string;
+    columns: {
+        className?: string;
+        expandableSections: {
+            header?: string;
+            iconType?: string;
+            iconColor?: string;
+            rows: {
+                header?: string;
+                rows: {
+                    accessor: React.ReactNode | string;
```

Comment:
Isn't this sometimes a function?  I guess if it's a function, then it's gotta be a function that acts as a component, so `React.ReactNode` probably works.

Reply:
I believe this to be right after looking at DataGroupRow.

Reply:
I trust you!  Just checking! :-)